app/helpers/application_helper.rb

```diff
@@ -381,13 +381,8 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
-  def documentation_button(section = nil)
-    url = if section
-            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}"
-          else
-            "http://www.theforeman.org/documentation.html##{SETTINGS[:version].short}"
```

Comment:
We've lost this conditional, which is used on the dashboard.  I'd say actually replace the link to the manual on the about page with the same link to the main documentation index.. it's a better entry point.